packages/@lwc/template-compiler/src/config.ts

```diff
@@ -39,20 +34,13 @@ export interface ResolvedConfig {
     format: Format;
 }
 
-const DEFAULT_CONFIG: ResolvedConfig = {
-    secure: false,
-    experimentalDynamicDirective: false,
-    experimentalComputedMemberExpression: false,
-    format: 'module',
-};
-
 const AVAILABLE_OPTION_NAMES = new Set([
     'secure',
```

Comment:
Is `secure` no longer being used? If so, we should also remove this.

Reply:
This is not used anymore, however, removing the `secure` property from the Set would result in a breaking change. So I decided to keep it there for now, until the next major release.